src/app/WebViewImpl.qml

```diff
@@ -699,6 +699,14 @@ WebView {
         }
 
     onFullScreenRequested: function(request) {
+        if (webview.isFullSceeen) {
+          webview.zoomController.refresh();
+        }
+        else {
+          // twice because of QTBUG-84313
+          webview.zoomFactor = 1.0;
+          webview.zoomFactor = 1.0;
```

Comment:
Why does zoom need to be reset to 1 when leaving fullscreen? Is this bypassing zoomController?

Reply:
actually it is set to 1 when entering fullscreen. (see https://github.com/ubports/morph-browser/issues/360)
If the current state is fullscreen, it is about to leave fullscreen mode.

Maybe the following would improve the readability ? 
```
onFullScreenRequested: function(request) {
        if (request.toggleOn) {
            // twice because of QTBUG-84313
            webview.zoomFactor = 1.0;
            webview.zoomFactor = 1.0;
        }
        else {
            webview.zoomController.refresh();
        }
        request.accept();
    }
```

Reply:
the zoom factor of the zoom controller remains unchanged, only in fullscreen mode it goes to 1.0
So it is bypassed while in fullscreen mode